onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -344,6 +352,9 @@ enum MLAS_CONV_ALGORITHM {
     MlasConvAlgorithmGemmDirect,
     MlasConvAlgorithmExpandThenGemm,
     MlasConvAlgorithmExpandThenGemmSegmented,
+#if defined(MLAS_TARGET_WASM)
```

Comment:
Hi, I think it's better to use the macro "MLAS_TARGET_WASM_SCALAR" here instead of "MLAS_TARGET_WASM" to make it align with code where using enum "MlasConvAlgorithmDepthwise".
Otherwise, it will throw error: 
convolve.cpp:929:21: error: enumeration value 'MlasConvAlgorithmDepthwise' not handled in switch [-Werror,-Wswitch] switch (Algorithm) {}
Please let me know if you have different ideas. Thanks.

Reply:
This is intented for future. And in other logic, currently only handle MLAS_TARGET_WASM_SCALAR.